snapshots/output/syntax/src/structural-type.ts

```diff
@@ -65,7 +65,7 @@
 //     ^^^^^^^^^^^^ definition syntax 1.0.0 src/`structural-type.ts`/FeatureFlags#
 //     documentation ```ts\ntype FeatureFlags\n```
 //                      ^^^^^^^^ definition syntax 1.0.0 src/`structural-type.ts`/FeatureFlags#typeLiteral13:darkMode.
-//                      documentation ```ts\n(property) darkMode: () => void\n```
+//                      documentation ```ts\n(property) darkMode: true\n```
```

Comment:
This is a serious regression! Need to look into it better, I think I know what's going on, and fixing this will probably fix the other `method/property` regression